src/vcpkg/binarycaching.cpp

```diff
@@ -1547,29 +1637,35 @@ namespace
             {
                 if (segments.size() < 2)
                 {
-                    return add_error("expected arguments: binary config 'files' requires at least a path argument",
+                    return add_error(std::move(msg::format(msgExpectedArguments)
+                                                   .append(msg::format(msgBinaryConfig, msg::config = "files"))
+                                                   .append(msgRequiresPathArgument)),
```

Comment:
To ensure the localization team has enough context to properly localize messages, we try to avoid factoring messages into many small pieces and assuming that they composite like English.

In this case, instead of individually localizing the messages "expected arguments: ", "binary config '{config}' ", and "requires at least a path argument", it would be better to use a single message:

`"expected argument in binary source '{binary_source}': a path is required"`